packages/tab/package.json

```diff
@@ -10,15 +10,16 @@
   "module": "dist/esm/index.js",
   "scripts": {
     "build": "run-s build:js build:cjs build:css",
-    "build:cjs": "babel --root-mode upward src --out-dir dist/cjs",
+    "build:cjs": "babel --root-mode upward src --out-dir dist/cjs && cp src/react/index.d.ts dist/cjs/react/index.d.ts && cp src/index.d.ts dist/cjs/index.d.ts",
```

Comment:
I don't like having to do this, not sure if there's a better way.

Reply:
Yeah, this is surprisingly manual.

How do other projects include their typings?

What if we allowed src/**/*.d.ts via npmignore?

Reply:
I don't think this is a common use case. The recommendation is to only include typings directly in the library if the library is written in TS[1]. (In the case that it's written in TS, you would have `tsc` emit the `.d.ts` files for you in the right place.) It seems like the main reason for this is that if the types are inacurrate/incomplete, you can iterate on the types independently without needing to cut a release of the library.

1: https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html

Reply:
Allowing `src/**/*.d.ts` in npmignore could be an option if we can rely on the `types` property to point to the right place.